app/assets/stylesheets/geoblacklight/modules/item.scss

```diff
@@ -7,6 +7,7 @@
 
   #viewer-container {
     [data-map="item"] {
+      border: 1px solid #ddd;
```

Comment:
Instead of hard-coding the color, maybe there's an appropriate bootstrap variable?

Reply:
👍 There isn't an exact match but I updated it to something close enough.